app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+
+from models import Actor
+
+class ActorManager:
+    def __init__(self):
+        self._connection =sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+# CREATE - C
+    def create(self, first_name: str, last_name:str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name,)
+        )
+        self._connection.commit()
+
+
+# RETRIEVE - R
+    def all(self) -> list:
+        actors.cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [
+            Actor(*row) for row in actors_cursor
+        ]
+
+# UPDATE - D
+    def update(self, id_to_update: int,
+               first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
+            " SET first_name = ? , last_name = ? "
+            " WHERE id = ? ",
+            (first_name, last_name, id_to_update,)
+        )
+        self._connection.commit()
+
+# DELETE - D
+    def delete(self, id_to_delete: int) -> None:
+        self._connection.execute(
+            f"DELETE FROM {self.table_name} WHERE id = ?",
+            (id_to_delete,)
+        )
+        self._connection.commit()
```

Comment:
Consider adding a method to close the database connection when it is no longer needed. This will help prevent potential resource leaks.